sirepo/template/cortex_sql_db.py

```diff
@@ -178,3 +229,18 @@ def _optional(v):
             ),
         ),
     )
+
+
+@contextlib.contextmanager
+def _session(qcall):
+    s = sirepo.sim_data.get_class("cortex")
```

Comment:
Probably should just use SIM_DATA.

Reply:
We could pass _SIM_DATA from the template.cortex to cortex_sql_db, but it would need to be on each call.

Reply:
use same header as template.cortex to create _SIM_DATA